src/lints/pub_api_sealed_trait_became_unconditionally_sealed.ron

```diff
@@ -0,0 +1,55 @@
+SemverQuery(
+    id: "pub_api_sealed_trait_became_unconditionally_sealed",
+    human_readable_name: "Public API sealed trait became unconditionally sealed",
+    description: "A trait that was previously only sealed to public API consumers is now unconditionally sealed. 
+                  This change prevents all external implementations, including from related first-party crates 
+                  that might have relied on private API access",
+    required_update: Major,
+    lint_level: Deny,
+    reference_link: Some("https://rust-lang.github.io/api-guidelines/future-proofing.html#sealed-traits-protect-against-downstream-implementations-c-sealed"),
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        public_api_sealed @filter(op: "=", value: ["$true"])
+                        unconditionally_sealed @filter(op: "!=", value: ["$true"])
+                        importable_path {
+                            path @output @tag
+                            public_api @filter(op: "=", value: ["$true"])
+                        }
+                    }
+                }
+            }
+            current {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+                        unconditionally_sealed @filter(op: "=", value: ["$true"])
+                        name @output
+                        importable_path {
+                            path @filter(op: "=", value: ["%path"])
+                            public_api @filter(op: "=", value: ["$true"])
+                        }
+                        span_: span @optional {
+                            filename @output
+                            begin_line @output
+                            end_line @output
+                        }
```

Comment:
nit: Please take a look at some of the other lints' queries and try to match the style and formatting of queries. The fact that this query is so dense and has no whitespace means it's quite difficult to read.